scss/base/buttons.scss

```diff
@@ -250,3 +250,27 @@ a.text-link.small {
   font-size: $font-size-sm;
   padding: 18px 40px;
 }
+
+.btn-default-dark,
+.btn-default-dark.active {
+  text-transform: uppercase;
+  box-shadow: none;
+  background-color: $gray-base-30;
+  border-color: $gray-base-30;
+
+  &:active {
+    box-shadow: none;
+    border-color: $gray-base-30;
+  }
+
+  &:hover {
+    background-color: $gray-base-30;
+  }
+}
+
+.btn-secondary.active {
```

Comment:
Let's discuss this as well - maybe we can stick with the default styling for `.btn-secondary`